decidim-core/app/services/decidim/activity_search.rb

```diff
@@ -27,6 +27,7 @@ def base_query
               .where(organization: options.fetch(:organization))
 
       query = query.where(user: options[:user]) if options[:user]
+      query = query.where(resource_type: options[:resource_name]) if options[:resource_name]
```

Comment:
why do we use `:resource_name` instead of `:resource_type` ?

Reply:
I had to use `:resource_name` from the options hash instead of `:resource_type` to avoid corrupting the filter, but the attribute is named `resource_type`, so the query has `.where(resource_type:` instead of `.where(resource_name:`